bird_cloud_gnn/radar_dataset.py

```diff
@@ -197,12 +221,19 @@ def load(self):
         self.max_distance = info["max_distance"]
         self.min_neighbours = info["min_neighbours"]
 
+    def cache_dir(self):
+        if self.data_folder is None:
+            directory = self.save_dir
+        else:
+            directory = os.path.dirname(self.data_folder)
```

Comment:
I noticed that if the `data_folder` is passed without a `/` at the end, this breaks. So I think it's better to write the following:

```suggestion
        elif os.path.isdir(self.data_folder):
            directory = self.data_folder
        elif os.path.isfile(self.data_folder):
            directory = os.path.dirname(self.data_folder)
        else:
            raise ValueError("Missing input. Either self.data_folder or self.input_data needs to be defined.")
```